### DISCUSSION TITLE
Failure with missing logs

### DISCUSSION THREAD

**Discussion by thewtex** [community/community#26478] [Actions]
<p>A Docker-based build step is failing and missing logs:</p>
<p><a href="https://github.com/InsightSoftwareConsortium/ITKAnisotropicDiffusionLBR/pull/40/checks?check_run_id=597920227" rel="nofollow noopener">https://github.com/InsightSoftwareConsortium/ITKAnisotropicDiffusionLBR/pull/40/checks?check_run_id=597920227</a></p>
<p>How should the logs be obtained or how can progress be made?</p>

**Comment by Yanjingzhu** [comment]
<p>Hi thewtex,</p>
<p>Sorry for letting you surferring this issue. I have found an existing issue in GitHub internal channel, engineering team is investigating on this issue. It might take some time before I get back to you. Appreciate your patience.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/thewtex">@thewtex</a> , </p>
<p>Your issue is caused by that the run used all disk space on hosted VM and cause the runner can’t finish its job.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1f65404ff2a55e767ae878688dcd349f52180858.png" data-download-href="/uploads/short-url/4tJJlcx6STHNs872o9KT7GNwjNu.png?dl=1" title="No space left.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181097951-d0b2d587-ff33-4735-b88a-8d21a2c56d95.png" alt="No space left.png" data-base62-sha1="4tJJlcx6STHNs872o9KT7GNwjNu" width="690" height="384" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/1f65404ff2a55e767ae878688dcd349f52180858_2_690x384.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/1f65404ff2a55e767ae878688dcd349f52180858_2_1035x576.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/1/1f65404ff2a55e767ae878688dcd349f52180858.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1f65404ff2a55e767ae878688dcd349f52180858_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">No space left.png</span><span class="informations">1189×662 37.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Ubuntu hosted runner provide a 14GB ssd disk space for jobs to use. There is an existing issue for hosted runner out of disk space problem <a href="https://github.com/actions/virtual-environments/issues/709" rel="nofollow noopener">https://github.com/actions/virtual-environments/issues/709</a>  </p>

**Comment by thewtex** [comment]
<p><span class="mention">@yanjingzhuthank</span> you such for for the tip and link! I look into reducing the build size and follow the issues with Ubuntu runners.</p>
